README.md

```diff
@@ -205,7 +205,7 @@ of the event loop, before the next paint).
 it does by default.
 * `update(changedProperties)`: Updates the element. Setting properties inside this
 method will *not* trigger another update.
-  * `render()`: Returns a `lit-html` TemplateResult (e.g. <code>html\`Hello ${world}\`</code>)
+* `render()`: Returns a `lit-html` TemplateResult (e.g. <code>html\`Hello ${world}\`</code>)
```

Comment:
I think this indentation was intentional by @sorvell to try to show that render is called as a part of LitElement's update implementation. Either way, I hope this is cleared up as we move to real docs.